code/modules/mining/mine_items.dm

```diff
@@ -278,153 +278,60 @@
 	icon_state = "capsule"
 	icon = 'icons/obj/mining.dmi'
 	w_class = 1
+	origin_tech = "engineering=3;bluespace=3"
+	var/template_id = "shelter_alpha"
+	var/datum/map_template/shelter/template
 	var/used = FALSE
 
+/obj/item/weapon/survivalcapsule/proc/get_template()
+	if(template)
+		return
+	template = shelter_templates[template_id]
+	if(!template)
+		throw EXCEPTION("Shelter template ([template_id]) not found!")
+		qdel(src)
```

Comment:
By the way, this proc should probably return 1 or 0 depending on whether the template could be loaded, so the procs that call it can return instead of immediately runtiming because `template` stays null.

Also, might be nice if it showed players a message before it winked out of existence.